srv/_modules/cluster.py

```diff
@@ -193,31 +236,45 @@ def storage_device_config():
         metadata_device.append(f"/dev/disk/by-id/dm-name-{device_list[0]}")
         metadata_field = f"cluster/{node}/storage/metadata_device".format(node)
         provisioner.pillar_set(metadata_field, metadata_device)
-        
-        data_device = [f"/dev/disk/by-id/dm-name-{device}" for device in device_list[1:]]
+
+        data_device = [
+            f"/dev/disk/by-id/dm-name-{device}"
+            for device in device_list[1:]
+        ]
         data_field = f"cluster/{node}/storage/data_devices"
         provisioner.pillar_set(data_field, data_device)
 
-    if (len(provisioner.pillar_get("cluster/srvnode-1/storage/data_devices"))
-        != len(provisioner.pillar_get("cluster/srvnode-2/storage/data_devices"))):
+    node1_data_devices_cnt = len(
```

Comment:
I would prefer to assign return value and then use `len` in conditional, it will be a bit more handy in case of debugging and locals dumping

Reply:
Done.